src/servers/src/postgres/types/error.rs

```diff
@@ -0,0 +1,517 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_error::status_code::StatusCode;
+use pgwire::error::ErrorInfo;
+use snafu::Snafu;
+use strum::{AsRefStr, Display, EnumIter, EnumMessage};
+
+#[derive(Display, Debug, PartialEq)]
+#[allow(dead_code)]
+enum ErrorSeverity {
+    #[strum(serialize = "INFO")]
+    Info = 0,
```

Comment:
I think we don't need to provide numerical representation for this enum. The postgres protocol doesn't define it